workspace-automation.md

```diff
@@ -0,0 +1,35 @@
+# Workspace Automation
+
+Over time, we've established various workspace automation to boost our productivity and get repetitive chores out of our way.
+
+## Slack Slash Commands
+
+Our slash commands automation is driven by [legion](https://github.com/eqworks/legion), our group chatbot.
+
+### `/avail` - Checks who is where
+
+This command complements our `Dev Avail` Asana project. The project allows our team to provide opt-in transparency of whether they are available to be contacted, in the form of physical presence in our office, or virtual presence remotely, on vacation, or not available.
```

Comment:
I find 'opt-in transparency' is a little hard to understand. One suggestion is: 'The project contains each team member's availability whether they're in the office, working remote, on vacation or not available. Information is provided by each team member.'

Reply:
i think it would be a bit clearer if i could link back to the `communications` doc, which is not in a settled shape yet.

i'll do it like the following based on your suggestion:

> This command complements our `Dev Avail` Asana project. In this project, each team member specifies their availability status, with optional details. This allows us to check each others' availability status without interruption.